### ISSUE TITLE
Linkable categories feature request

### ISSUE THREAD

**Issue by rocchidavide** [codefog/contao-news_categories#52]
What about adding the possibility to set a destination page for every category? 
Imagine to have a news list in a page, and for every news its categories list. Could be useful to set a page for a specific category, for example that contains another news list module filtered only for the clicked category

**Comment by qzminski** [comment]
Implemented in 46e5fe0.

**Comment by qzminski** [comment]
To display the links in the news template use the below code:

```php
<?php if ($this->categories): ?>
<ul class="categories">
	<?php foreach ($this->categories as $category): ?>
	<li class="category_<?php echo $category['id']; ?>">
	
		<?php if ($category['href']): ?>
		<a href="<?php echo $category['href']; ?>" title="<?php echo $category['linkTitle']; ?>">
		<?php endif; ?>
	
		<?php echo $category['name']; ?>
		
		<?php if ($category['href']): ?>
		</a>
		<?php endif; ?>
		
	</li>
	<?php endforeach; ?>
</ul>
<?php endif; ?>
```
